lib/msal-node/src/retry/IHttpRetryPolicy.ts

```diff
@@ -13,4 +15,8 @@ export interface IHttpRetryPolicy {
         currentRetry: number,
         retryAfterHeader: number
     ): Promise<boolean>;
+
+    retryAfterMillisecondsToSleep(
```

Comment:
There is no need to expose this method on the interface, as only the retry policy uses it. Can you avoid exposing it on the interface?

The retry policy object has access to the "Retry-After" header and can compute the number of seconds to pause on its own. Doesn't that help?